admin_app/deployment/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  frontend-container:
+    image: ${DEMO_ACCOUNT}.dkr.ecr.af-south-1.amazonaws.com/aaq-demo-ecr-repository:frontend
+    command: >
+      npm start
+    restart: always
+    ports:
+      - 3000:3000
+    logging:
+      driver: awslogs
+      options:
+        awslogs-group: /ecs/frontend-task-demo
+        awslogs-region: af-south-1
+        awslogs-stream-prefix: frontend
```

Comment:
Can we call this `admin_app` (or `admin_app_container`)?

Reply:
Changed in the Pipeline PR